packages/cli/openapi-parser/src/openapi/v3/hasIncompleteExample.ts

```diff
@@ -1,12 +1,25 @@
 import { EndpointExample } from "@fern-api/openapi-ir-sdk";
+import { RawSchemas } from "@fern-api/yaml-schema";
 
 // this means the example only includes code samples and should be autogenerated
 export function hasIncompleteExample(example: EndpointExample): boolean {
-    return (
-        example.request == null &&
-        example.response == null &&
-        example.queryParameters == null &&
-        example.pathParameters == null &&
-        example.headers == null
-    );
+    if (example.type === "full") {
+        return (
+            example.request == null &&
+            example.response == null &&
+            example.queryParameters == null &&
+            example.pathParameters == null &&
+            example.headers == null
+        );
+    } else {
+        const castExample = example.value as RawSchemas.ExampleEndpointCallSchema;
+        // throw new Error(`Here we are: ${JSON.stringify(castExample)}`);
```

Comment:
nit: we'll want to delete but makes sense why you added

Reply:
haha whoops